tests/Unit/ActionTesting.hpp

```diff
@@ -1680,6 +1690,26 @@ const auto& get_databox_tag(
       .template get_databox_tag<Tag>();
 }
 
+// @{
+/// Returns the `InboxTag` from the `DataBox` of the parallel component
```

Comment:
I'm confused - are inboxes in the databox? I had assumed they were a separate data structure.

Reply:
You're right, this was a copy-paste mistake. Fixed now :)